Paintroid/src/main/java/org/catrobat/paintroid/MainActivity.kt

```diff
@@ -279,8 +280,8 @@ class MainActivity : AppCompatActivity(), MainView, CommandListener {
     }
 
     public override fun onCreate(savedInstanceState: Bundle?) {
-        setTheme(R.style.PocketPaintTheme)
         super.onCreate(savedInstanceState)
+        installSplashScreen()
```

Comment:
in the [migration guide](https://developer.android.com/develop/ui/views/launch/splash-screen/migrate#best-practices) it is noted to call installSplashScreen() before super.onCreate().
Please change this